src/Socket.cpp

```diff
@@ -30,7 +30,7 @@
 #ifdef HAVE_LIBSSL
 // Copypasted from
 // https://wiki.mozilla.org/Security/Server_Side_TLS#Intermediate_compatibility_.28default.29
```

Comment:
Apparently keeping this page open without refreshing didn't show your comments, thanks for testing, breaking HexChat is undesirable AFAIK.